src/bp/core/services/nlu/entities-service.ts

```diff
@@ -0,0 +1,104 @@
+import * as sdk from 'botpress/sdk'
+
+import { GhostService } from '..'
+
+import * as CacheManager from './cache-manager'
+import { IntentService } from './intent-service'
+
+const ENTITIES_DIR = './entities'
+
+const DUCKLING_ENTITIES = [
+  'amountOfMoney',
+  'distance',
+  'duration',
+  'email',
+  'number',
+  'ordinal',
+  'phoneNumber',
+  'quantity',
+  'temperature',
+  'time',
+  'url',
+  'volume'
+]
+
+export class EntityService {
+  private intentService!: IntentService
+
+  constructor(private ghostService: GhostService) {}
+
+  public load(intentService: IntentService) {
+    this.intentService = intentService
+  }
+
+  private sanitizeFileName(name: string): string {
+    return name
+      .toLowerCase()
+      .replace(/\.json$/i, '')
+      .replace(/[\t\s]/gi, '-')
+  }
```

Comment:
Please move this to an `utils.ts` file, if we don't already have one somwehre. We can use the same for both entities and intents. Also, try to keep outside the class methods which doesn't call `this`